extensions/wikia/Chat2/ChatBanListSpecial.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @package MediaWiki
+ * @subpackage SpecialPage
+ * @author Piotr Molski <moli@wikia-inc.com> for Wikia.com
+ * @version: 1.0
+ */
+class ChatBanListSpecial extends SpecialRedirectToSpecial
+{
+
+	function __construct() {
+		parent::__construct( 'ChatBanList', 'chatbanlist' );
+	}
+
+	public function execute( $subpage ) {
+		global $wgOut, $wgJsMimeType, $wgResourceBasePath;
+
+		wfProfileIn( __METHOD__ );
+
+		$scriptSource = "{$wgResourceBasePath}/resources/wikia/libraries/jquery/datatables/jquery.dataTables.min.js";
+
+		$wgOut->setPageTitle( wfMessage( 'chatbanlist' )->text() );
```

Comment:
In subclasses of both `WikiaSpecialPageController` and `SpecialPage` you have access to a `RequestContext` instance so you can use `$this->getOutput()` and `$this->msg()` instead of globals.